tests/capture/test_nested_plxpr.py

```diff
@@ -159,3 +161,111 @@ def qfunc(wire):  # x is closure variable and a tracer
 
         assert len(q) == 1
         qml.assert_equal(q.queue[0], qml.adjoint(qml.RX(2.5, 2)))
+
+
+class TestCtrlQfunc:
+    """Tests for the ctrl primitive."""
+
+    def test_operator_type_input(self):
+        """Test that an operator type can be the callable."""
+
+        def f(x, w):
+            return qml.ctrl(qml.RX, 1)(x, w)
+
+        plxpr = jax.make_jaxpr(f)(0.5, 0)
+
+        with qml.queuing.AnnotatedQueue() as q:
+            out = jax.core.eval_jaxpr(plxpr.jaxpr, plxpr.consts, 1.2, 2)
+
+        assert f(0.5, 0) is None
+        assert out == []
```

Comment:
Following @dime10's suggestion.
```suggestion
        assert out is None
```

Reply:
`jax.core.eval_jaxpr` always returns a list of tensorlike's.  But we do have `f(0.5, 0) is None` tested on the line above.